omnibus/files/private-chef-cookbooks/private-chef/recipes/redis_lb.rb

```diff
@@ -132,15 +132,8 @@
 
     redis.pipelined do
       # Now we're clear to repopulate from configuration.
-      unless banned_ips.nil?
```

Comment:
What was the intent of the banned_ips?

Reply:
To block the IPs irrespective of maintenance mode on or off